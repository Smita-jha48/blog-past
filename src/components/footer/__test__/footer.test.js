import { render } from '@testing-library/react';
import Footer from '../footer';

describe('Footer', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Footer/>);
    expect(asFragment()).toMatchSnapshot();
  });
});