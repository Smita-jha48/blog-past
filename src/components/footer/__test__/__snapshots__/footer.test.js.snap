// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <small>
      Â©artifact.com 2019
    </small>
  </footer>
</DocumentFragment>
`;
